react/features/aeternity/utils.js

```diff
@@ -50,3 +50,21 @@ export function isInIframe() {
 export function isWalletJWTSet(state) {
     return state['features/base/jwt'].jwt;
 }
+
+/**
+ * Returns query string and hash concatnated.
+ *
+ * @param {string} urlString - Url string.
+ * @returns {string}
+ */
+export function getQueryStringAndFragment(urlString) {
+    try {
+        const url = new URL(urlString);
+        const search = url.search;
+        const hash = url.hash;
+
+        return `${search}${hash}`;
+    } catch (error) {
+        console.log(error);
+    }
+}
```

Comment:
I remember that we have some same functionality in one of jitsi URL utls

Reply:
@Chima1707 sorry, I mean about that utl, it's a bit different
https://github.com/aeternity/jitsi-meet/blob/d67f2e971c86211c8bdf1c2eed811d416f9c8e60/react/features/base/util/parseURLParams.js#L5-L15